_posts/en/newsletters/2021-06-16-newsletter.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Bitcoin Optech Newsletter #153'
+permalink: /en/newsletters/2021/06/16/
+name: 2021-06-16-newsletter
+slug: 2021-06-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter celebrates the lock in of the taproot soft fork,
+describes a draft BIP for improving transaction
+privacy by varying the fields used to implement anti fee sniping, and
+features an article about the challenges of combining transaction
+replacement with payment batching.  Also included are our regular
+sections with announcements of new software releases and release
+candidates, plus notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **🟩  Taproot locked in:** the [taproot][topic taproot] soft fork and
+  related changes specified in BIPs [340][bip340], [341][bip341], and
+  [342][bip342] were locked in by signaling miners last weekend.
+  Taproot will be safe to use after block 709,632, which is expected in
+  early or mid November.  The delay gives time for users to upgrade
+  their nodes to a release (such as Bitcoin Core 0.21.1) that will
+  enforce taproot's rules, ensuring that funds received to taproot
+  scripts after block 709,632 are safe even if there's a problem with
+  miners.
+
+    Developers are encouraged to start [implementing taproot][taproot
+    uses] so they can
+    be ready to take advantage of greater efficiency, privacy, and
+    fungibility as soon as the activation is complete.
+
+    Readers celebrating the lock in of taproot may also wish to read a
+    [short history][wuille taproot] of taproot's origins and history by
+    developer Pieter Wuille.
+
+- **BIP proposed for wallets to set nSequence by default on taproot transactions:**
+  Chris Belcher [posted][belcher post] a draft BIP to the Bitcoin-Dev
+  mailing list suggesting an alternative way wallets can implement [anti
+  fee sniping][topic fee sniping].  The alternative method would enhance
+  the privacy and fungibility of transactions made by single-sig users,
+  [multisignature][topic multisignature] users, and users of certain
+  contract protocols such as taproot-enabled LN or advanced
+  [coinswaps][topic coinswap].
+
+    Anti fee sniping is a technique some wallets implement to discourage
+    miners from trying to steal fees from each other in a way that would
+    reduce the amount of proof of work expended on securing Bitcoin and
+    limit users' ability to rely on confirmation scores.  All wallets
```

Comment:
Perhaps
```suggestion
    limit users' ability to rely on initial confirmations.  All wallets
```

Reply:
The examples here focus on a single block reorg, but it can get much much worse.